authors/apps/authentication/tests/test_authentication.py

```diff
@@ -0,0 +1,87 @@
+from django.urls import reverse
+from django.contrib.auth import get_user_model
+from rest_framework import status
+from rest_framework .test import APITestCase, APIClient
+
+
+class UserManagerTestCase(APITestCase):
+    """Usermanager TestCase."""
+
+    def setUp(self):
+        self.client = APIClient()
+
+        self.user_registration = {
```

Comment:
@mwinel Use test fixture. Have a sync with @3Nakajugo so you can proceed with this. I left a similar comment on her pr.